### DISCUSSION TITLE
Missing `goog` dependencies (ClojureScript, Reagent/React)

### DISCUSSION THREAD

**Discussion by petar-jr** [storybookjs/storybook#17168] [Help]
I am trying to write stories in ClojureScript, for components written in ClojureScript. The components are written using `Reagent`, which means they are almost `React` components. However, the application has pretty much custom format and storybook is of course not able to recognize its type as React application, so I guess it's not able to do any nice thing that it is usually able to do for normal React applications. 

Anyway, the application is built using `figwheel` to "compile" ClojureScript files to JavaScript, and I am configuring Storybook to look at those generated JavaScript files containing stories, and Storybook is finding them OK. However, the problem is that the JavaScript files with stories are invoking methods from `goog` package, which does not get included when running Storybook, so Storybook is not able to load those stories, because for it "goog is not defined".  

Is there a way to configure Storybook to somehow "bootstrap" the stories and include the necessary libraries, like `goog`?

**Comment by shilman** [comment]
You can do global stuff in `.storybook/preview.js`, so I'm wondering if you could do something hacky like:

```js
// .storybook/preview.js
import goog from 'goog';
window.goog = goog;
```
